### ISSUE TITLE
Support EXCEPT as a sql keyword

### ISSUE THREAD

**Issue by dvsweeney9** [knex/knex#5350]
# Environment

Knex version: 2.3.0
Database + version: PostgresQL 12.x
OS: Ubuntu 14.3-1.pgdg18.04+1

# Feature discussion / request

1. Explain what is your use case
I want to be able to use `EXCEPT` as a keyword in postgres.  https://www.techonthenet.com/postgresql/except.php#:~:text=The%20PostgreSQL%20EXCEPT%20operator%20is,statement%20will%20define%20a%20dataset.
It's used similar to `UNION` and `INTERSECT` which are supported by knex. It has the form `(subquery) EXCEPT (subquery)`

2. Explain what kind of feature would support this
This would be supported by anyone who needs to use `EXCEPT` in their SQL query.

3. Give some API proposal, how the feature should work
This would be very similar to the way `UNION` and/or `INTERSECT` are operated. I'd expect it to look like:
`.except([*queries], [wrap])`
For example:
```
knex.select('*')
  .from('users')
  .whereNull('last_name')
  .except([
    knex.select('*').from('users').whereNull('first_name')
  ])
```
which would parse to something like:
```
SELECT * FROM users WHERE last_name IS NULL
EXCEPT
SELECT * FROM users WHERE first_name IS NULL
```
and would return rows where last_name is null and first_name is not null. 
This is a simple example, but has some very realistic use cases. 
For my purposes, this is a required functionality for a very complex query we have.

**Comment by dvsweeney9** [comment]
This is functionality that is blocking my project so I put up a PR if somebody could review.
https://github.com/knex/knex/pull/5357

**Comment by OlivierCavadenti** [comment]
Merged

**Comment by OlivierCavadenti** [comment]
Released in [2.5.0](https://github.com/knex/knex/releases/tag/2.5.0)
